test/asyncapi.yaml

```
@@ -1,4 +1,4 @@
-asyncapi: '2.1.0'
+asyncapi: '2.2.0'
```

Taking this opportunity to update it to the last (as per today) version: `2.3.0`.
```suggestion
asyncapi: '2.3.0'
```